### PR TITLE
implement POST & GET image in backend

### PR DESCRIPTION
**Pull request by uwhuan**

### CODE REVIEW

**Comment by danielmerken** [comment]
LGTM :shipit: :ship: :rocket:

**Review by danielmerken — APPROVED** [review]
:shipit: :shipit: :shipit:

**Comment by bparadee** [comment]
depending on how frontend wants to access our data, it might make sense to include the image set/get in the create/getSubmissions functions so that we avoid making multiple calls.

**Comment by kourbou** [comment]
Testing class looks good, but we should also think about making a test SQL server at some point to test our SQL code...

File: backend/src/main/java/edu/uw/cs403/plantmap/backend/controllers/ImageController.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        this.server = server;
+    }
+
+    public Object uploadImg(Request request, Response response){
+        try{
+            // parse submission id from URL
+            int sub_id = Integer.parseInt(request.params("id"));
+            server.updateSubmission(sub_id, request.bodyAsBytes());
+            response.type("text/html");
+            return "Successfully upload image";
```
Comment (kourbou) [inline backend/src/main/java/edu/uw/cs403/plantmap/backend/controllers/ImageController.java]: Not sure about the reply here, maybe it's best to return the `sub_id`, using response type `text/plain`?

**Review by kourbou — APPROVED** [review]

File: backend/src/main/java/edu/uw/cs403/plantmap/backend/controllers/ImageController.java (hunk 1 above)
Comment (uwhuan) [reply backend/src/main/java/edu/uw/cs403/plantmap/backend/controllers/ImageController.java]: done~

**Review by uwhuan — COMMENTED** [review]
reviewed

**Pull request merged**